src/progress-steps/examples-list.js

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-env node */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+module.exports = {
+  DEFAULT: 'Dots example',
+  NUMBERED: 'Numbered example',
```

Comment:
Could you also add an example with overrides?

Reply:
I added an override example that includes a custom color